MapboxCoreNavigation/MBRouteController.h

```diff
@@ -55,10 +55,20 @@ extern const NSNotificationName MBRouteControllerDidPassVisualInstructionPointNo
 typedef NSString *MBRouteControllerNotificationUserInfoKey NS_EXTENSIBLE_STRING_ENUM;
 
 /**
- A key in the user info dictionary of a `Notification.Name.MBRouteControllerProgressDidChange` or `Notification.Name.RouteControllerDidPassSpokenInstructionPoint` notification. The corresponding value is a `RouteProgress` object representing the current route progress.
+ A key in the user info dictionary of a `Notification.Name.routeControllerProgressDidChange`, `Notification.Name.routeControllerDidPassVisualInstructionPoint`, or `Notification.Name.routeControllerDidPassSpokenInstructionPoint` notification. The corresponding value is a `RouteProgress` object representing the current route progress.
```

Comment:
`Notification.Name.routeControllerProgressDidChange` is a generated Swift symbol so jazzy won't have access to it. Document according to Obj-C symbols here.